binaries/install.py

```diff
@@ -15,14 +16,14 @@ def install():
         channel_dir = os.path.abspath(os.path.join(REPO_ROOT, "binaries", "conda", "output"))
         conda_cmd = f"conda install --channel {channel_dir} -y torchserve torch-model-archiver"
         print(f"## In directory: {os.getcwd()} | Executing command: {conda_cmd}")
-        install_exit_code = os.system(conda_cmd)
+        install_exit_code = subprocess.run(conda_cmd)
```

Comment:
https://github.com/pytorch/serve/pull/1913#pullrequestreview-1149544750

Reply:
Thank you for pointing out. Updated the PR to fix the exit code issue as well as add shell=True in subprocess.run